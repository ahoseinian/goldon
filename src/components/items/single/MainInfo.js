import React from 'react';
import styled from 'styled-components'
import {color} from '../../theme'
import CodeAndPrice from './CodeAndPrice'

const ItemName = styled.h1 `
  color:${color.grey};
  text-align: center;
  line-height: 3rem;
  padding: 1rem .5rem;
`
const MainInfo = ({item}) => (
  <div>
    <ItemName>{item.name}</ItemName>
    <CodeAndPrice item={item}/>
  </div>
);

MainInfo.propTypes = {
  item: React.PropTypes.object.isRequired
};

export default MainInfo;
