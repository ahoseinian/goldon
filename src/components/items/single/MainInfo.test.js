import React from 'react'
import {shallow} from 'enzyme'
import MainInfo from './MainInfo'
import toJson from 'enzyme-to-json'


describe('MainInfo', () => {
  it('renders', () => {
    const wrapper = shallow(<MainInfo item={{name:'test item'}} />)
    expect(toJson(wrapper)).toMatchSnapshot();
  })
})
