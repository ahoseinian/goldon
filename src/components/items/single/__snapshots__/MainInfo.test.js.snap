exports[`MainInfo renders 1`] = `
<div>
  <styled.h1>
    test item
  </styled.h1>
  <CodeAndPrice
    item={
      Object {
        "name": "test item",
      }
    } />
</div>
`;
